CHANGELOG.md

```diff
@@ -1,15 +1,46 @@
 # Changelog
 
-## 0.3.0
+## 0.4.0
 
 ### Added
 
+- Support for `#[default = "..."]` in struct methods [[#436]](https://github.com/extendr/extendr/pull/436)
```

Comment:
Should we explain that this applies to the parameters of methods? This way someone may think it is something applied to the method itself. Also, it does not really explains why we add this.

Reply:
Probably yes. Feel free to provide a better explanation (preferably as a suggestion)